shell/components/ResourceCancelModal.vue

```diff
@@ -99,32 +99,27 @@ export default {
     margin: 0 10px;
   }
 
-  .v--modal-box {
```

Comment:
I see more `v--modal-box` around, should we be worried?

Reply:
The final two instances are associated with `shell/components/PromptModal.vue` (thoroughly tested and used throughout Dashboard) & `shell/dialog/RotateCertificatesDialog.vue` (uses `PromptModal`, the associated `rotate-modal` class appears to be unused). 

I removed both of those instances, because they were really unused.